py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2934,6 +2934,50 @@ def aten_ops_flip(
     )
 
 
+def zero_diag_size_validator(node: Node) -> bool:
+    input_shape, offset, dim1, dim2 = (
+        node.args[0].meta["tensor_meta"][0],
+        node.args[1],
+        node.args[2],
+        node.args[3],
+    )
+
```

Comment:
Do we need to call `get_positive_dim` like you did in the ops.py?

Reply:
You are definitely right. The `get_positive_dim` function is also needed here. Thank you!